modules/apps/foundation/portal-search/portal-search-test-util/src/main/java/com/liferay/portal/search/test/util/indexing/BaseIndexSearcherTestCase.java

```diff
@@ -143,15 +135,39 @@ private void _assertPagination(int start, int end, int expectedSize)
 		searchContext.setEnd(end);
 		searchContext.setStart(start);
 
-		Hits hits = indexSearcher.search(searchContext, getDefaultQuery());
+		IdempotentRetryAssert.retryAssert(
```

Comment:
Thanks @arboliveira ! Let me know once you have it done, I'd like to see it goes away :)